kythe/go/test/testutil/testutil.go

```diff
@@ -171,6 +164,14 @@ func TrimmedEqual(got, want []byte) (bool, string) {
 	return diff == "", diff
 }
 
+func caller(up int) (file string, line int) {
```

Comment:
Maybe for a future commit: I think we can probably get rid of this if we use `t.Helper()` judiciously here too.